src/m365/pa/commands/app/app-export.ts

```diff
@@ -0,0 +1,238 @@
+}
+
+class PaAppExportCommand extends PowerPlatformCommand {
+  private static readonly sleepInterval: number = 5000;
+
+  public get name(): string {
+    return commands.APP_EXPORT;
+  }
+
+  public get description(): string {
+    return 'Exports the specified Power App';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        packageDescription: typeof args.options.packageDescription !== 'undefined',
+        packageCreatedBy: typeof args.options.packageCreatedBy !== 'undefined',
+        packageSourceEnvironment: typeof args.options.packageSourceEnvironment !== 'undefined',
+        path: typeof args.options.path !== 'undefined'
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-i, --id <id>'
+      },
+      {
+        option: '-e, --environment <environment>'
+      },
+      {
+        option: '-n, --packageDisplayName <packageDisplayName>'
+      },
+      {
+        option: '-d, --packageDescription [packageDescription]'
+      },
+      {
+        option: '-c, --packageCreatedBy [packageCreatedBy]'
+      },
+      {
+        option: '-s, --packageSourceEnvironment [packageSourceEnvironment]'
+      },
+      {
+        option: '-p, --path [path]'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (args.options.id && !validation.isValidGuid(args.options.id)) {
+          return `${args.options.id} is not a valid GUID`;
+        }
+
+        if (args.options.path && !fs.existsSync(path.dirname(args.options.path))) {
+          return 'Specified path where to save the file does not exist';
+        }
+
+        return true;
+      }
+    );
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      const packageLink = await this.getPackageLink(args, logger);
+      //Replace all illegal characters from the file name
+      const illegalCharsRegEx = /[\\\/:*?"<>|]/g;
+      const filename = args.options.packageDisplayName.replace(illegalCharsRegEx, '_');
+
+      if (this.verbose) {
+        logger.logToStderr(`Name of the Power Apps: ${filename}`);
+      }
+
+      const requestOptions: CliRequestOptions = {
+        url: packageLink,
+        // Set responseType to arraybuffer, otherwise binary data will be encoded
+        // to utf8 and binary data is corrupt
+        responseType: 'arraybuffer',
+        headers: {
+          'x-anonymous': true
+        }
+      };
+
+      const file = await request.get<string>(requestOptions);
+
+      const path = `${args.options.path ? args.options.path : './'}${args.options.packageDisplayName}.zip`;
+
+      fs.writeFileSync(path, file, 'binary');
+      if (!args.options.path || this.verbose) {
```

Comment:
Let's remove arts.options.path as a condition here. Whether you specify a path or not, you want to know that it got saved and where. Some confirmation is okay here.